src/components/Card.tsx

```
@@ -0,0 +1,174 @@
+
+  const {
+    cardExpirationDate,
+    handleCardExpirationDateChange,
+    isError: isCardExpirationDateError,
+    errorMessage: cardExpirationDateErrorMessage,
+  } = useCardExpirationDate();
+
+  const {
+    cardCompany,
+    handleCardCompanyChange,
+    isError: isCardCompanyError,
```

ESLint

[error] 'isCardCompanyError' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)